release_lib.sh

```diff
@@ -34,7 +34,7 @@ function make_control {
 	do
 		case "$DAEMON_PKG" in
 		'clickhouse-server' )
-			add_daemon_impl clickhouse-server-base '' 'clickhouse-server binary'
+			add_daemon_impl clickhouse-server-base 'adduser' 'clickhouse-server binary'
```

Comment:
I read this just now:
```
useradd is a low level utility for adding users. On Debian, administrators should usually use adduser(8) instead.
```

What if people are installing debian package on non-debian system (this way is used in practice)? Dependencies are cost too much... I am not sure that `adduser` gives enough comparing to `useradd`. BTW, I call these kind of programs "trash-scripts".